Lib/test/test_concurrent_futures/executor.py

```diff
@@ -205,3 +220,16 @@ def test_free_reference(self):
             for _ in support.sleeping_retry(support.SHORT_TIMEOUT):
                 if wr() is None:
                     break
+
+    def test_swallows_falsey_exceptions(self):
+        # see gh-132063: Prevent exceptions that evaluate as falsey
+        # from being ignored.
+        # Recall: `x` is falsey if `len(x)` returns 0 or `bool(x)` returns False.
+
+        msg = 'boolbool'
+        with self.assertRaisesRegex(FalseyBoolException, msg):
+            self.executor.submit(raiser, FalseyBoolException, msg).result()
```

Comment:
This is for a possible follow-up but this kind of call makes me wonder whether the base class shouldn't have some `def submit_and_fetch(self)` method that does this.

Reply:
Does this method return directy the result ?

Reply:
There is a follow-up with the `TracebackException`class which swallows `__cause__` and `__context__` content of falsey exceptions. 
I will post a new issue about this.